src/core_modules/capture-core/components/Pages/MainPage/WorkingListsCommon/hooks/useWorkingListsCommonStateManagementOffline.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+// $FlowFixMe
+import { useSelector, shallowEqual } from 'react-redux';
+
+const useList = (storeId: string) => {
+    const listState = useSelector(({ workingLists, workingListsListRecords, workingListsColumnsOrder }) => ({
+        eventRecords: workingListsListRecords[storeId],
+        recordsOrder: workingLists[storeId] && workingLists[storeId].order,
+        customColumnOrder: workingListsColumnsOrder[storeId],
+    }), shallowEqual);
```

Comment:
Why is `shallowEqual`  better option here?

Reply:
We could do `useSelector` x 3, but opted to use shallowEqual instead.

Reply:
Thats nice 👍